### ISSUE TITLE
Graph tails at 0h/s at 0min mark normal?

### ISSUE THREAD

**Issue by MoneroChan** [M5M400/supportxmr-gui#12]
Hi, 
Just checking if this is just affecting me or everyone else?

-  I noticed when you login, the graph displays 0h/s constantly (at the right tail end).

- Please see:  https://imgur.com/a/jPTnuBN

Graph says "0.0 H/s Your Hash" (even after 5 mins refreshing it remains at 0.0 H/s )

the "5 min ago" mark is not affected and displays correct hashrate.

Has this been fixed in the latest release? or is this intended?
I'm not sure if it's got something to do with my system time. i set the system time to +10 mins into the future, but even resetting my system time to  +0min didn't seem to fix it.

(old platform doesn't have this issue.)

cheers,

**Comment by Thunderosa** [comment]
Good catch! I had worked on scenarios where the local was behind the API clock because I encountered it, but it totally skipped my mind to fix when the local time is ahead. Bonehead move on my part, I'll patch that and get it in the next release. Thanks!

**Comment by MoneroChan** [comment]
No worries, cheers
